swan-lake/learn-the-platform/configure-observe/configure-ballerina-programs/provide-values-to-configurable-variables.md

```diff
@@ -89,5 +89,9 @@ The mapping of Ballerina types to TOML types can be explained through the follow
 | record with record field | <code>type  Food  record {</code><br/>    <code>string  name;</code><br/>    <code>int  cal;</code><br/><code>};</code><br/><code>type  Diet  record {</code><br/>    <code>Food  food;</code><br/>    <code>int  age;</code><br/><code>};</code><br/>  <code>configurable   Diet  input = ?;</code>  | TOML table - nested              | `[input]`<br/>  `age = 20`<br/> `food.name = "carrot"`<br/> `food.cal = 41`<br/>                                                            |
 | record[]                 | <code>type  Person  record {</code><br/>    <code>string  name;</code><br/>    <code>int  age;</code><br/><code>};</code><br/>  <code>configurable   Person[]  peope = ?;</code>                                                                                                                      | Array of TOML tables             | `[[people]]`<br/>  `name = "John"`<br/> `age = 45`<br/> `[[people]]`<br/>  `name = "Jack"`<br/> `age = 32`                                  |
 | table                    | <code>configurable   table &lt;map&lt;string&gt;&gt; users = ?; </code>                                                                                                                                                                                                                               | Array of TOML tables             | `[[users]]`<br/> `name = "Tom"`<br/> `occupation = "Software Engineer"`<br/> `[[users]]`<br/> `name = "Harry"`<br/> `occupation = "Doctor"` |
+| table[]                  | <code>configurable   table &lt;map&lt;string&gt;&gt;[] userTeams = ?; </code>                                                                                                                                                                                                                         | 2D Array of inline-TOML tables   | `userTeams = [[{name = "Tom", team = "Dev"}, {name = "Harry", team = "Dev"}], [{name = "Anna", team = "Finance"}]]`                         |
+
+>**Note:** The `()` type is supported in a configurable variable only if it has optional configuration. In a 
+> configurable variable of record type, it is only supported if the `()` typed fields has default values.
```

Comment:
```suggestion
<br/>

>**Note:** The `()` type is supported in a configurable variable only if it has the optional configuration. In a 
record type configurable variable, it is only supported if the `()` typed fields have default values.
```

Reply:
Shall we also add an entry in the table for the `()` type to show an example for what is described in the note?
It may be not clear what is meant by the optional configuration.

Reply:
We can't add an example for this because we can't configure nil values through TOML syntax. That's why we accept only cases with default values. 
If we have an entry with sample `configurable () nilValue = ();`, there will not be a entry for `TOML type` column.